### PR TITLE
Reformatted code and more changes.

### PR DESCRIPTION
**Pull request by unckleg**
- Reformatted JS code to follow Vue.js & newest ES standards/recommendations.
- Changed mutable vars to immutable by changing them from **_'let'_** to **_'const'_**.
- Added .idea folder to _.gitignore_

### CODE REVIEW

File: public/js/about.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
 import createPage from './create-page.js';
 
-export let showoff = (name) => console.log(name, ' showing off');
```
Comment (unckleg) [inline public/js/about.js:3]: No need for using let declaration here.

File: public/js/about.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+export const showoff = (name) => console.log(name, ' showing off');
 
-let about = createPage(
```
Comment (unckleg) [inline public/js/about.js:5]: No need for using let declaration here.

File: public/js/app.js [hunk 3]
```diff
@@ -1,51 +1,51 @@
-import about      from './about.js';
```
Comment (unckleg) [inline public/js/app.js:1]: Formatting changes.

File: public/js/create-page.js [hunk 4]
```diff
@@ -1,27 +1,27 @@
-let createPage = (name, data = {}, methods = {}, updated = () => {}) => Vue.component('page-' + name, {
```
Comment (unckleg) [inline public/js/create-page.js:1]: Formatting changes.

**Review by unckleg — COMMENTED** [review]
- Formatted JS code from 4 spaces to 2 spaces.

**Comment by samsonasik** [comment]
@unckleg merged, thank you ;)

**Pull request merged**